tests/console/btrfs_autocompletion.pm

```diff
@@ -29,6 +31,13 @@ sub compare_commands {
 sub run {
     select_console 'root-console';
 
+    # On JeOS 'bash-completion' is not expected to be present. On general
+    # SLES installation it is. Thus on JeOS we have to enable it manually.
+    if (is_jeos) {
+        zypper_call('in bash-completion');
+        assert_script_run('source $(rpmquery -l bash-completion | grep bash_completion.sh)');
+    }
+
```

Comment:
@DimStar77 maybe this is something that you want to go for in TW as well when you don't want to install bash-completion by default